tutorials/quantize_vit/run.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Run bfloat16 version
+TORCH_LOGS='graph_breaks,recompiles' python run_vit_b.py
+
+# Run dynamic quantized version
+TORCH_LOGS='graph_breaks,recompiles' python run_vit_b_quant.py
+
+# Store the output code for further inspection
+TORCH_LOGS='output_code' python run_vit_b.py 2> bfloat16_code
+TORCH_LOGS='output_code' python run_vit_b_quant.py 2> quant_code
```

Comment:
I think what people might expect to see here is the fused quant and dequant in the generated code so we just add some comments to the checked in code so people can inspect

Reply:
I'll try to add more pictures and also isolate the kernel. It's much clearer from the traces.